src/jit/hwintrinsicxarch.cpp

```diff
@@ -188,7 +199,6 @@ static InstructionSet lookupInstructionSet(const char* className)
         return InstructionSet_LZCNT;
     }
 
-    unreached();
     return InstructionSet_ILLEGAL;
```

Comment:
I think we have an `assert` that needs to be handled here: https://github.com/dotnet/coreclr/blob/486aa8c3003f07fb1bbb460046748e6802150496/src/jit/hwintrinsicxarch.cpp#L51

and an `unreached` here: https://github.com/dotnet/coreclr/blob/486aa8c3003f07fb1bbb460046748e6802150496/src/jit/hwintrinsicxarch.cpp#L102

I don't think these will be hit today though, since nothing in `Vector64` is `[Intrinsic]` yet.

Reply:
I think the first one,`assert(isa != InstructionSet_ILLEGAL); `, can be removed.

Reply:
I think it can be removed and short-circuited, since we know the for loop is useless.

Reply:
Agree.